src/chrome/content/rules/ponominalu.ru.xml

```diff
@@ -0,0 +1,126 @@
+<!--
+help.ponominalu.ru ¹
+www.nnov.ponominalu.ru ¹
+ns1.ponominalu.ru ¹
+ns3.ponominalu.ru ¹
+ns4.ponominalu.ru ²
+one.ponominalu.ru ²
+kvm.one.ponominalu.ru ⁴
+three.ponominalu.ru ³
+kvm.three.ponominalu.ru ⁴
+two.ponominalu.ru ³
+kvm.two.ponominalu.ru ⁴
+ponominalu.ru different content
+www.ponominalu.ru different content
+airfest.ponominalu.ru different content
+arh.ponominalu.ru different content
+cheb.ponominalu.ru different content
+dud.ponominalu.ru different content
+ekb.ponominalu.ru different content
+kaluga.ponominalu.ru different content
+kost.ponominalu.ru different content
+krd.ponominalu.ru different content
+kzn.ponominalu.ru different content
+m.ponominalu.ru different content
+nnov.ponominalu.ru different content
+ns2.ponominalu.ru different content
+omsk.ponominalu.ru different content
+rnd.ponominalu.ru different content
+sochi.ponominalu.ru different content
+spb.ponominalu.ru different content
+tomsk.ponominalu.ru different content
+tver.ponominalu.ru different content
+vdm.ponominalu.ru different content
+volg.ponominalu.ru different content
+yola.ponominalu.ru different content
```

Comment:
Could you double check the 'different content' issues?
I think they are working now.